### DISCUSSION TITLE
Artist art folder

### DISCUSSION THREAD

**Discussion by bfd69** [navidrome/navidrome#2755] [Ideas]
Hello
is it possible to have a folder with all my artist art (jpg/png etc) ?
this is something that was discussed in #394 but i lost track of it
at some point there was a proposition for a fixed folder that would contain artists art in the form of {artist}.* which i would love to see implemented. 
This is the main functionality that prevents me from migrating to navidrome as i have near a thousand of artists with just one or two song each, and it would be long to create a folder each.

is it doable ? is it planned ?
again i found navidrome very nice and i'd love to dump ampache for it, i especially love the compilation support.

may be others would love this functionality ?
Best regards

**Comment by dweymouth** [comment]
How would you deal with different artists with the same name?

**Reply by bfd69** [reply]
Hello
First, I wish you a happy new year!
thanks for taking some of your time, i really appreciate.
I think that a folder artist will solve 98% of the case, and for the few cases where you have artists with same name or with names not compatible with the host filesystem.
Ill just make use of the artist art resolution priority order so that artist art folder is fetched before external sources and after id3 tag song art.

The order in my settings would be : 

Album folder,song folder,Local artist folder,lastfm,spotify

Like this i make a big folder with my long time gathered artist art, and for exeptional cases ill just create a folder for one song or two OR add artist art in the song as an id3tag.

Thanks again for hearing me, feel free to contact me if its not clear.

**Comment by bfd69** [comment]
That would be a problem even for art folder based artists
The goal is to centralise arts in one folder for 90% of my artists

It will be complicated to manage one folder per artist when you have +1000 songs with different artist each like singles...
